components/scanners/osv-scanner/scanner/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
```

Comment:
why is this needed? can't we invoke the osv scanner without having an entrypoint?